features/admin.administrators.v1/pages/administrators.tsx

```diff
@@ -36,7 +36,7 @@ import { administratorConfig } from "@wso2is/admin.extensions.v1/configs/adminis
 import FeatureGateConstants from "@wso2is/admin.feature-gate.v1/constants/feature-gate-constants";
 import { getAgentConnections } from "@wso2is/admin.remote-userstores.v1/api/remote-user-stores";
 import { AgentConnectionInterface } from "@wso2is/admin.remote-userstores.v1/models/remote-user-stores";
-import { getRoleById, searchRoleList } from "@wso2is/admin.roles.v2/api/roles";
+import { getRoleById, getRoleByIdV3, searchRoleList } from "@wso2is/admin.roles.v2/api/roles";
```

Comment:
adding v3 api methods in roles.v2 module may make it harder for new devs to understand/find code in the long run, IMO. Shall we declare a new module `admin.roles.v3` and move the new methods there? WDYT?

cc: @brionmario @DonOmalVindula

Reply:
If we create a role.v3 module, it will lead to code duplication. In my opinion, we don't need to create a new module for each API version. For now, I’ve added a method specifically for Role V3 and will update the logic to invoke the V3 API later.